README.md

```diff
@@ -157,8 +153,6 @@ Feel free to check the [issues page](../../issues/).
 
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
```

Comment:
- [ ] This PR should be invalidated by the previous reviewer because the support section is empty with placeholder you should never leave your readme file empty with placeholders so that it will be pro